Makefile

```diff
@@ -180,6 +180,10 @@ else
 endif
 export KUSTOMIZE
 
+install: operator-sdk
+	@echo "Installing operators"
+	hack/install-ocs-operator.sh
```

Comment:
As we have already a hack file install-ocs.sh, how about we call it install-all/install-all-operators.sh ?

Reply:
I will remove all those install scripts once this goes in. We will need to make a change in the release repo and after that we can remove that.